public/stylesheets/index.css

```diff
@@ -310,6 +310,7 @@ nav .login.button {
   position: relative;
   margin-top: 56px;
   flex-direction: column;
+  width:725px;
```

Comment:
The footer already has a fixed width, so I think the best value for this would be `100%` to just fill all the remaining area.

Reply:
the problem was without setting a width the width was changing because the width was set to 100%. Im not sure if you fixed it already but that's why the change is there.

Reply:
we can just delete this if you fixed it already

Reply:
Seems like this change got merged in with some other branch. I'll close this one then.